src/api/CertificateAuthority.ts

```diff
@@ -0,0 +1,68 @@
+import axios, { AxiosError, AxiosInstance, AxiosResponse } from "axios";
+import ServiceVersion from "@/api/helpers/models/ServiceVersion";
+import APIResponse from "./helpers/models/APIResponse";
+import APIError from "./api_models/errors/APIError";
+
+export default class CertificateAuthority {
+    _axios: AxiosInstance;
+
+    constructor() {
+        const instance = axios.create({
+            baseURL: process.env.VUE_APP_CA_BASE_URL,
+            headers: {
+                Accept: "*/*",
+            },
+            withCredentials: false,
+        });
+        delete instance.defaults.headers.common.Authorization;
+
+        this._axios = instance;
+    }
+
+    async getAllCACertificates(): Promise<string[]> {
+        const caNames = ["org0", "org1", "org2", "tls"];
```

Comment:
Do we need to hardcode the names? They should be stored in a global const at the beginning of the file 
Fetching them could be some nice refactoring in the future

Reply:
Correct, I thought of a way to fetch them. I will implement that on another branch (issue) and will keep the constants for now